packages/react-storage/src/components/StorageBrowser/composables/ActionStart.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ButtonElement } from '../context/elements';
+import { CLASS_BASE } from '../views/constants';
+
+export interface ActionStartProps {
+  onClick?: () => void;
```

Comment:
Ah missed this last round but can we associate this with the intended behavior instead? For example, a custom ActionStart component may not be a button but instead.. register a keystroke to start the action.
```suggestion
  onStart?: () => void;
```